core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -37,6 +37,8 @@ class EmbeddedZookeeper() {
   factory.startup(zookeeper)
   val port = zookeeper.getClientPort()
 
+  println("Starting embedded zookeeper on port " + port + " using directories " + snapshotDir + "," + logDir)
```

Comment:
@guozhangwang I haven't been able to recreate the problem locally. If the print is added now, it can be removed once we see a test failure in the builds with the logging.